public/pages/about-me.html

```diff
@@ -0,0 +1,110 @@
+                </div>
+                <ul class="mobile-menu-ul">
+                <li><a class="mobile-menu-ul-li-a link" data-replace="About" href='./about-me.html'><span class="link-text">About</span></a></li>
+                <li><a class="mobile-menu-ul-li-a link main-program-link" data-replace="Program" href='#main-program'><span class="link-text">Program</span></a></li>
+                <li><a class="mobile-menu-ul-li-a link" data-replace="Join" href='#'><span class="link-text">Join</span></a></li>
+                <li><a class="mobile-menu-ul-li-a link" data-replace="Sponsor" href='#footer'><span class="link-text">Sponsor</span></a></li>
+                <li><a class="mobile-menu-ul-li-a link" data-replace="News" href='#'><span class="link-text">News</span></a></li>
+                </ul>
+            </div>
+        </header>
+    <section class="about">
+        <h1 class="about-text-1"><span class="about-text-1-span hidden-if-large">"Hello! Sharing Fans"</span><span class="main-text">KPOP Conference 2022</span></h1>
+        <p class="about-text-2">The Korea Remixed K-pop Conference explores the historic, cultural, and political significance of South Korea&apos;s signature music. The conference will feature panels with IU experts, visiting scholars, and IU graduate and undergraduate students.</p>
+        <div class="about-contact">
+            <span class=".about-contact-span">Please Contact us per Email for any further questions</span>
+            <span class=".about-contact-span">about KPOP conference 2022!!</span>
+            <span class=".about-contact-span">ccsumit2015@cckorea.org</span>
+        </div>
+    </section>
+
+    <section class="logo-section">
+        <h2 class="logo-header"><span class="main-text">KPOP Conference 2022</span></h2>
+        <div class="underline">&nbsp;</div>
+        <p class="logo-p">Our logo is decided on 2021 May. It was drawn by Hein Tay Zar through the local competition in Feb 2021</p>
+        <div class="logo-about">
+            <img src="../resources/kpop_black.png" width="80" class="contact-me-logo" alt="logo">
+        </div>
+    </section>
+
+    <section class="past-projects">
+        <h3 class="past-projects-h3">See Our Pasts</h3>
+        <div class="underline">&nbsp;</div>
+        <p class="past-projects-p">Let's see our pasts Projects and prepare for incoming conference. Let's Take a look and explore toghther for fun and inspire ideas for them</p>
+        <ul class="past-projects-ul">
+            <li class="past-projects-ul-li"><div class="past-projects-ul-li-div"><span class="past-project-year">2021</span><span class="past-project-text">See our past projects in Seoul</span></div></li>
+            <li class="past-projects-ul-li"><div class="past-projects-ul-li-div"><span class="past-project-year">2020</span><span class="past-project-text">See our past projects in Seoul</span></div></li>
+        </ul>
```

Comment:
- [ ] What the reviewer meant here in this section is that you should match the design provided, in the design there is a red overlay on the pictures, kindly add it, your design is looking like this right now:

  ![image](https://user-images.githubusercontent.com/20567503/196688772-d6fbba98-2d03-4b2e-a20d-22b55f9c2567.png)
  
  And it should be like this 😁: 
  
  ![image](https://user-images.githubusercontent.com/20567503/196688833-0ea28cb4-0301-4f64-88b2-422bfdd1e355.png)